_posts/2022-02-21-lanl.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+title:  "NumFOCUS signs agreement with LANL to improve nonlinear support in JuMP"
+date:   2022-02-21
+categories: [announcements]
+---
+
+The [JuMP Steering Committee](/pages/governance/#steering-committee) is pleased
+to announce that we, through [NumFOCUS](https://numfocus.org), have signed an
+agreement with [Los Alamos National Laboratory (LANL)](https://www.lanl.gov) to
+improve nonlinear support in JuMP.
+
+The agreement runs until September 2023.
+
+At a high-level, the agreement seeks to improve nonlinear support in JuMP on two
+key fronts:
+ * improving the automatic differentiation performance in JuMP by adding the
+   ability to use different automatic differentiation backends
+ * improving the modeling ergonomics of JuMP by adding the ability to create
+   nonlinear expressions using the `@constraint` and `@objective` macros, and
+   identifying ways to add vector-valued nonlinear functions.
+
+In addition, there is scope to work on and improve other aspects of nonlinear
+programming in JuMP.
+
+We are well aware that JuMP 1.0 is nearing completion (https://github.com/jump-dev/JuMP.jl/issues/2564).
```

Comment:
I don't think this paragraph is necessary. Instead, maybe advertise the monthly developer call and/or monthly nonlinear call for anyone interested to participate and ask questions about our plans.

Reply:
Updated.